README.md

```diff
@@ -425,11 +428,19 @@ Write the `contents` to `filepath` at the given random access position. When `po
 
 Moves the file located at `filepath` to `destPath`. This is more performant than reading and then re-writing the file data because the move is done natively and the data doesn't have to be copied or cross the bridge.
 
+Note: Overwrites existing file in Windows.
+
+### `copyFolder(srcFolderPath: string, destFolderPath: string): Promise<void>`
+
+Copies the contents located at `srcFolderPath` to `destFolderPath`.
+
+Note: Windows only. This method is recommended when directories need to be copied from one place to another.
```

Comment:
The second part of the sentence doesn't seem to add much... maybe describe instead why you would use this as opposed to (...)? is this about appcontainer restrictions?

Reply:
the reason copyDirectory was added was because using copyFile to copy over the contents of a folder would take a long long time. It would take 40 seconds to copy over a 60 MB directory with a large number of files using copyFile recursively when it could just be copy-pasted within a fraction of a second in file explorer. It appeared that a lot of conversions needed to be done between calling a function via JavaScript to it being executed, and I am not as familiar with the inner workings of React Native for Windows as I should be. This method is a workaround that allows folders like that 60 MB directory mentioned earlier to be copied much faster than they would be without it.

Reply:
Thanks @avmoroz! this is good feedback for the nodejs folks - we have definitely heard (and seen) nodejs is a lot slower on Windows than linux. I filed https://github.com/nodejs/node/issues/36158 - if you have any more data or can leave more context on that bug, that'd be super appreciated too